_typos.toml

```diff
@@ -0,0 +1,23 @@
+[default.extend-identifiers]
```

Comment:
Oh, this is going to be painful potentially. Is there no way to disable checking long strings that are clearly not real words?

Reply:
Great idea, I'll see if it's possible.
I agree it's easier and less error-prone than maintaining a list of exceptions.

Reply:
I was able to write few regexes, which allow not to hardcode this list